### PR TITLE
New schema test

### PR DESCRIPTION
**Pull request by hascar**
Implemented new schema for tests
The tests are green here: https://ci.openquake.org/job/zdevel_oq-platform2/253/

### CODE REVIEW

File: local_settings.py.tmpl [hunk 1]
```diff
@@ -112,6 +112,12 @@ ALLOWED_DOCUMENT_TYPES = [
 ]
 MAX_DOCUMENT_SIZE = 100
 
+STANDALONE_APPS = (
```
Comment (nastasi-oq) [inline local_settings.py.tmpl]: Like for old platform, please, do not duplicate this list into `INSTALLED_APPS` but concatenate `STANDALONE_APPS` to it: https://github.com/gem/oq-platform/blob/master/openquakeplatform/openquakeplatform/local_settings.py.template#L292

Comment (hascar) [reply local_settings.py.tmpl]: Fixed here: https://github.com/gem/oq-platform2/pull/23/commits/96509f853fb9d64aca825ab47c46b30dbff80311

**Comment by nastasi-oq** [comment]
Review on going by @nastasi-oq

File: openquakeplatform/test/versions_test.py [hunk 2]
```diff
@@ -21,6 +21,11 @@ def page_app_versions_test(self):
         # macthing if the version of applications
         # content numbers and points
         search_version_app = pla.xpath_finduniq(
-            "//table[@class='versions_table']//tr/td[../td[normalize-space(text())='IPT']][normalize-space(text())!='IPT']")
+            "//table[@class='versions_table']//tr/"
+            "td[../td[normalize-space(text())"
+            "='IPT']][normalize-space(text())!='IPT']")
         vers = search_version_app.text
-        self.assertRegexpMatches(vers, '^[0-9]+\.', msg=("version [%s] not valid for IPT app" % vers))
+        self.assertRegexpMatches(vers, '^[0-9]+\.', msg=("version [%s]"
```
Comment (nastasi-oq) [inline openquakeplatform/test/versions_test.py]: Please, add (a real) newline before `"version` and concatenate the rest of the string.

**Pull request merged**